lib/ReactViews/Analytics/ParameterEditor.jsx

```diff
@@ -27,6 +27,177 @@ const ParameterEditor = React.createClass({
         previewed: React.PropTypes.object
     },
 
+    parameterTypeConverters: [
```

Comment:
Does this actually work?  We want this property to be static, so that it can be modified by an application without a specific instance handy.  I suppose you can do that by accessing `ParameterEditor.prototype.parameterTypeConverters` but that's ugly.  Probably better to remove this from here and instead add `ParameterEditor.parameterTypeConverters = ...` just before the return at the end of this file.